runtime/autoload/health/nvim.vim

```diff
@@ -3,9 +3,20 @@ let s:bad_responses = [
             \ 'unable to parse python response',
             \ 'unable to parse',
             \ 'unable to get pypi response',
-            \ 'unable to get neovim executable'
+            \ 'unable to get neovim executable',
+            \ 'unable to find neovim version'
```

Comment:
For example, there were times when something could be set to `unknown` in multiple places. You might say `let neovim_version = "unknown"`, and then check stuff, and then get past a set of checks, and then set it again. I just made it so the error was explicit what coulnd't be found.